src/main/java/de/muenchen/dave/services/MesswerteMessquerschnittService.java

```diff
@@ -0,0 +1,75 @@
+package de.muenchen.dave.services;
+
+import de.muenchen.dave.geodateneai.gen.api.MesswerteMessquerschnittApi;
+import de.muenchen.dave.geodateneai.gen.model.GetMesswerteMessquerschnittRequest;
+import de.muenchen.dave.geodateneai.gen.model.MesswerteIntervallMessquerschnittDto;
+import de.muenchen.dave.geodateneai.gen.model.MesswerteTagesaggregatMessquerschnittDto;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class MesswerteMessquerschnittService {
+
+    private final MesswerteMessquerschnittApi messwerteMessquerschnittApi;
+
+    /**
+     * Diese Methode ruft die angefragten MesswerteIntervalle eines Messquerschnitts aus der
+     * Geodaten-Eai ab.
+     *
+     * @param messstelleId Id der angefragten Messstelle
+     * @param von required Zeitpunkt der Daten.
+     * @param bis optional Ende eines Zeitraums.
+     * @param tagesTyp Typ des Tages
+     * @return Liste der gefundenen MesswerteIntervalle pro Tag
+     */
+    public List<MesswerteIntervallMessquerschnittDto> ladeMesswerteIntervall(final long messstelleId, final String von, final String bis,
+            final String tagesTyp) {
+        final GetMesswerteMessquerschnittRequest request = new GetMesswerteMessquerschnittRequest();
+        request.setMessstelleId(messstelleId);
```

Comment:
Das mit der Auflösung der messstellenId zu zwei messquerschnitteIds müssen wir später im Backend und in der EAI nachziehen, wenn wir mehr Anforderungen haben. 
Das Frontend könnte die Daten für eine Messstelle anfordern, das Backend übersetzt das in zwei Messquerschnitte und die EAI fragt den FeatureService nach den Messwerten für beide Messquerschnitte ab. Was hältst Du davon?

Reply:
Hab das als Kommentar eingefuegt und einfach eine Dummy-Aufloesung eingebaut.